pkg/distributor/distributor_test.go

```diff
@@ -4859,33 +4862,6 @@ func TestHandleIngesterPushError(t *testing.T) {
 			require.Equal(t, testData.expectedIngesterPushError.errorCause(), ingesterPushErr.errorCause())
 		})
 	}
-
-	// Ensure that the errors with no status (e.g., context.Canceled,
```

Comment:
These test cases have been removed because they are not realistic: errors returned by `IngesterClient.Push()` will never be a plain GoLang errors, but gRPC errors. For example, `IngesterClient.Push()` will never return `context.Canceled` or `context.DeadlineExceeded`, but equivalents to `status.New(codes.Canceled, context.Canceled.Error())` or `status.New(codes.DeadlineExceeded, context.DeadlineExceeded.Error())`.